### ISSUE TITLE
Augmentation layers only active during training!

### ISSUE THREAD

**Issue by innat** [keras-team/tf-keras#320]
**Description of issue (what needs to be changed):**.
**Correct links**. https://keras.io/guides/preprocessing_layers/

As it said, the following layers apply random augmentation transforms to a batch of images. They are only active **during training**.

[tf.keras.layers.RandomCrop](https://keras.io/api/layers/preprocessing_layers/image_augmentation/random_crop#randomcrop-class)
[tf.keras.layers.RandomFlip](https://keras.io/api/layers/preprocessing_layers/image_augmentation/random_flip#randomflip-class)
[tf.keras.layers.RandomTranslation](https://keras.io/api/layers/preprocessing_layers/image_augmentation/random_translation#randomtranslation-class)
[tf.keras.layers.RandomRotation](https://keras.io/api/layers/preprocessing_layers/image_augmentation/random_rotation#randomrotation-class)
[tf.keras.layers.RandomZoom](https://keras.io/api/layers/preprocessing_layers/image_augmentation/random_zoom#randomzoom-class)
[tf.keras.layers.RandomHeight](https://keras.io/api/layers/preprocessing_layers/image_augmentation/random_height#randomheight-class)
[tf.keras.layers.RandomWidth](https://keras.io/api/layers/preprocessing_layers/image_augmentation/random_width#randomwidth-class)
[tf.keras.layers.RandomContrast](https://keras.io/api/layers/preprocessing_layers/image_augmentation/random_contrast#randomcontrast-class)

So, in the model, if we use these layers, they will be **active** (i.e. `model.fit` / `model(..., training=True)` during training but will be **inactive** when we call `model.evaluate` and `model.predict` / `model(..., training=False)`. A sample model would be 

```python
model = keras.Sequential(
   [
      tf.keras.layers.RandomHeight,
      tf.keras.layers.RandomContrast,
      tf.keras.layers.Conv2D,
      tf.keras.layers.Dense
      ...
    ]
)
```  

Queries

- What if I want the augmentaiton layer be active in the prediction phase (i.e for making test-time-augmentaiton), using `model.predict`? We can use `model(..., training=True)` but (a). It's best suited for small amount of data, and (b). some other function (`dropout`) might be active. 
- Already checked it. https://github.com/keras-team/keras/blob/e6784e4302c7b8cd116b74a784f4b78d60e83c26/keras/backend.py#L451-L456
